src/add_lable.py

```diff
@@ -0,0 +1,58 @@
+import pandas as pd
+from dotmap import DotMap
+from github import Github
+from github import Auth
+from github.Repository import Repository
+
+ORIGINAL_ISSUES_FILE = "../res/issues.json.zip"  # ../res/ o res/
+LABELED_ISSUES_FILE = "../res/labeled_issues.json"
+SECONDS_TO_LABEL = 60  # Secondi entro i quali l'etichetta deve essere stata aggiunta
+REPOSITORY = "microsoft/vscode"
+AUTH_TOKEN = "INSERT_YOUR_TOKEN"
+
+repo: Repository
+testing = False  # False to run all the issues, otherwise the number of firsts issues to run
+
+
+def init_github():
+    auth = Auth.Token(AUTH_TOKEN)
```

Comment:
Remove the AUTH_TOKEN constant, use the same approach for token retrieval as in the:
[pull_request.py](https://github.com/lorenzovarese/automated-bug-triaging/blob/main/src/pull_issues.py).
Normally it is better to pass it as an environmental variable